sklearn/kernel_ridge.py

```diff
@@ -6,11 +6,11 @@
 
 import numpy as np
 
+from .base import BaseEstimator, RegressorMixin
+from .metrics.pairwise import pairwise_kernels
+from .linear_model.ridge import _solve_cholesky_kernel
 from .utils import check_X_y
-from sklearn.base import BaseEstimator, RegressorMixin
-from sklearn.metrics.pairwise import pairwise_kernels
-from sklearn.linear_model.ridge import _solve_cholesky_kernel
-from sklearn.utils.validation import check_is_fitted
```

Comment:
Tangential issue: I am confused now about whether or not to avoid relative imports in tests. For some reason, I thought absolute imports were preferred. Although I do agree that by a stroke of bad luck, the absolute import may cause the import of an entirely different package.

@agramfort why did you ask for absolute imports [here](https://github.com/nilearn/nilearn/pull/219#discussion_r19062716) ?